mantle/cmd/kola/testiso.go

```diff
@@ -53,46 +54,78 @@ var (
 
 	instInsecure bool
 
-	nopxe bool
-	noiso bool
-
-	scenarios []string
+	tests []string
 
 	pxeAppendRootfs bool
 	pxeKernelArgs   []string
 
 	console bool
 
-	addNmKeyfile bool
+	addNmKeyfile	 bool
+	enable4k	 bool
+	enableMultipath	 bool
+	enableUefi	 bool
+	enableUefiSecure bool
+
+	// The iso-as-disk tests are only supported in x86_64 due the hybrid partition table.
+	tests_x86_64 = []string {
+		"iso-as-disk.bios",
+		"iso-as-disk.bios.4k",
+		"iso-as-disk.uefi",
+		"iso-as-disk.uefi-secure",
+		"iso-live-login.bios",
+		"iso-live-login.bios.4k",
+		"iso-live-login.uefi",
+		"iso-live-login.uefi-secure",
+		"iso-offline-install.bios",
+		"iso-offline-install.bios.4k",
+		"iso-offline-install.bios.mpath",
```

Comment:
I think one thing we missed when doing the list of tests is that there's also `iso-install` that's available as an opt-in test not run by default. I think we don't want to lose that. Can we add an e.g. `iso-install.bios` test here to cover it? Eventually maybe we can make it optional like it was before so that it's not impossible to run it.